### PR TITLE
Rapid7 Nexpose - Fix Description

### PR DESCRIPTION
**Pull request by MichaelYochpaz**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Fix Rapid7 Nexpose pack description to include InsightVM for better clarity.

### CODE REVIEW

File: Packs/Rapid7_Nexpose/Integrations/Rapid7_Nexpose/README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-Rapid7's on-premise vulnerability management solution, Nexpose, helps you reduce your threat exposure by enabling you to assess and respond to changes in your environment real time and prioritizing risk across vulnerabilities, configurations, and controls.
-This integration was integrated and tested with version 6.6.103 of Rapid7 Nexpose
+Rapid7's vulnerability management solutions, Nexpose and InsightVM, help you reduce your threat exposure by enabling you to assess and respond to changes in your environment real time and prioritizing risk across vulnerabilities, configurations, and controls.
+This integration was integrated and tested with version 6.6.103 of Rapid7 Nexpose.
```
Comment (bakatzir) [inline Packs/Rapid7_Nexpose/Integrations/Rapid7_Nexpose/README.md:2]:
```suggestion
Rapid7's vulnerability management solutions, help you reduce your threat exposure by enabling you to assess and respond to changes in your environment real time and prioritizing risk across vulnerabilities, configurations, and controls.
This integration was integrated and tested with version 6.6.103 of Rapid7 Nexpose.
```

File: Packs/Rapid7_Nexpose/Integrations/Rapid7_Nexpose/Rapid7_Nexpose.yml [hunk 2]
```diff
@@ -25,7 +25,7 @@ configuration:
   name: proxy
   required: false
   type: 8
-description: Rapid7's on-premise vulnerability management solution, Nexpose, helps you reduce your threat exposure by enabling you to assess and respond to changes in your environment real time and prioritizing risk across vulnerabilities, configurations, and controls.
+description: Rapid7's vulnerability management solutions, Nexpose and InsightVM, help you reduce your threat exposure by enabling you to assess and respond to changes in your environment real time and prioritizing risk across vulnerabilities, configurations, and controls.
```
Comment (bakatzir) [inline Packs/Rapid7_Nexpose/Integrations/Rapid7_Nexpose/Rapid7_Nexpose.yml:28]:
```suggestion
description: Rapid7's vulnerability management solutions, help you reduce your threat exposure by enabling you to assess and respond to changes in your environment real time and prioritizing risk across vulnerabilities, configurations, and controls.
```

File: Packs/Rapid7_Nexpose/Integrations/Rapid7_Nexpose/Rapid7_Nexpose.yml [hunk 3]
```diff
@@ -25,7 +25,7 @@ configuration:
...
 display: Rapid7 Nexpose
```
Comment (bakatzir) [inline Packs/Rapid7_Nexpose/Integrations/Rapid7_Nexpose/Rapid7_Nexpose.yml:29]:
```suggestion
display: Rapid7 InsightVM
```

File: Packs/Rapid7_Nexpose/pack_metadata.json [hunk 4]
```diff
@@ -1,8 +1,8 @@
 {
     "name": "Rapid7 Nexpose",
-    "description": "Nexpose is Rapid7's on-premises option for vulnerability management, real-time exposures monitoring, and threat adaptation.",
+    "description": "Rapid7's vulnerability management solutions, Nexpose and InsightVM, help you reduce your threat exposure by enabling you to assess and respond to changes in your environment real time and prioritizing risk across vulnerabilities, configurations, and controls.",
```
Comment (bakatzir) [inline Packs/Rapid7_Nexpose/pack_metadata.json:3]:
```suggestion
    "description": "Rapid7's vulnerability management solution, help you reduce your threat exposure by enabling you to assess and respond to changes in your environment real time and prioritizing risk across vulnerabilities, configurations, and controls.",
```

File: Packs/Rapid7_Nexpose/pack_metadata.json [hunk 5]
```diff
@@ -1,8 +1,8 @@
 {
     "name": "Rapid7 Nexpose",
```
Comment (bakatzir) [inline Packs/Rapid7_Nexpose/pack_metadata.json:2]:
```suggestion
    "name": "Rapid7 InsightVM",
```

**Review by bakatzir — COMMENTED** [review]
Please add `Nexpose` in the keywords of the pack_metadata.

it seems as though Nexpose is re branded as InsightVM
https://www.rapid7.com/products/nexpose/insightvm-comparison/

AKA Demisto to XSOAR.

**Comment by MichaelYochpaz** [comment]
@bakatzir Should I also update the icon from Nexpose to InsightVM to match the rename?

**Comment by bakatzir** [comment]
@MichaelYochpaz yes. we usually use the portfolio company logo, and not the product. so please use `Packs/Rapid7_InsightIDR/Integrations/Rapid7_InsightIDR/Rapid7_InsightIDR_image.png`.

**Comment by ShirleyDenkberg** [comment]
@bakatzir @bziser Doc review completed.

**Review by bziser — APPROVED** [review]
Great 👍

**Pull request merged**